ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -111,6 +111,15 @@ handle('toggleFullscreenWindow', () => {
  */
 handle('closeWindow', () => app.quit());
 
+/**
+ * Return an object containing helper fields for determining what OS we're running on
+ */
+handle('checkOS', () => ({
+  isLinux: isLinux(),
```

Comment:
Nice!! I think, before we just used os checkings within `main` process, nice pattern to expose that to the rendered!!